src/common/math.h

```diff
@@ -238,6 +238,15 @@ union float_int {
   int k;
 };
 
+// a faster, vectorizable version of hypotf() when we know that there won't be overflow, NaNs, or infinities
+#ifdef _OPENMP
+#pragma omp declare simd 
+#endif
+static inline float dt_fast_hypotf(const float x, const float y)
+{
+  return sqrtf(x * x + y + y);
```

Comment:
Isn't this meant to be `sqrtf(x * x + y * y);`?

Reply:
Indeed ! Good catch I would probably have missed the error here as `*` and `+` are quite similar when read on diff.

Reply:
:blush: Fixed.